code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -272,7 +272,9 @@
 		M.current.visible_message("<span class='warning'>[M.current.name] lets out an ear piercing shriek!</span>", "<span class='warning'>You let out a loud shriek.</span>", "<span class='warning'>You hear a loud painful shriek!</span>")
 		for(var/mob/living/carbon/C in hearers(4, M.current))
 			if(C == M.current) continue
-			if(ishuman(C) && C:is_on_ears(/obj/item/clothing/ears/earmuffs)) continue
+			if(ishuman(C))
+				var/mob/living/carbon/human/H = C
+				if(H.is_on_ears(/obj/item/clothing/ears/earmuffs)||H.is_on_ears(/obj/item/device/radio/headset/headset_earmuffs)) continue
```

Comment:
Figure out a better way to do this than hardcoded copypaste.
eyeprot() is a good example.